src/site/content/en/blog/script-component/index.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: Optimizing third-party script loading in Next.js
+subhead: Understand the vision behind the Next.js’ Script component, which provides a built-in solution to optimize the loading of third-party scripts.
+date: 2022-02-23
+updated: 2021-02-23
```

Comment:
Since this is a new post, I'd leave out the `updated` portion of this front matter.